docs/using_python_client_with_hazelcast_imdg.rst

```diff
@@ -1526,99 +1712,42 @@ TIMESTAMP_WITH_TIME_ZONE datetime.datetime (with non-None tzinfo)
-evaluates to a result of type boolean. Any row that doesn’t satisfy the
-condition is eliminated from the result.
-
-**Sorting**
-
-You can use the standard SQL clauses ``ORDER BY``, ``LIMIT``, and ``OFFSET``
-to sort and limit the result set.
+Functions and Operators
+~~~~~~~~~~~~~~~~~~~~~~~
 
+Hazelcast supports logical and ``IS`` predicates, comparison and mathematical
+operators, and aggregate, mathematical, trigonometric, string, table-valued,
+and special functions.
 
-.. warning::
-
-    Note that, you must add sorted indexes to the map object’s fields to be
-    sorted by. For example, for the ``SELECT * FROM persons ORDER BY name ASC``
-    query, there has to be a sorted index on the ``name`` field as shown below:
-
-    .. code:: python
-
-        persons = client.get_map("persons")
-        persons.add_index(attributes=["name"], index_type=IndexType.SORTED)
-
-See the below examples for sorting.
-
-The following statement gets the top five employees ordered by the
-``first_name`` field and skipping the first three ones:
-
-.. code:: sql
-
-    SELECT
-        employee_id, first_name, last_name
-    FROM
-        employees
-    ORDER BY first_name
-    LIMIT 5 OFFSET 3;
-
-
-The following statement gets the top five employees with the highest salaries.
-
-.. code:: sql
-
-    SELECT
-        employee_id, first_name, last_name, salary
-    FROM
-        employees
-    ORDER BY salary DESC
-    LIMIT 5;
-
-**Unsupported Features**
-
-The following features are **not supported** and are planned for future releases:
+See the `Reference Manual
+<https://docs.hazelcast.com/hazelcast/latest/sql/expressions.html>`__
+for details.
 
-- ``GROUP BY`` / ``HAVING``
-- ``JOIN``
-- set operators (``UNION``, ``INTERSECT``, ``MINUS``)
-- subqueries (``SELECT ... FROM table WHERE x = (SELECT …)``)
+Source and Sink Connectors
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Expressions
-~~~~~~~~~~~
+SQL connectors are extensions that allow you to communicate with external
+systems such as databases, using SQL. These connectors are configured to read
+and write data in the most efficient way for their respective system.
 
-Hazelcast SQL supports logical predicates, `IS` predicates, comparison
-operators, mathematical functions and operators, string functions, and special
-functions.
+To use a connector, you must use the `CREATE MAPPING statement
+<https://docs.hazelcast.com/hazelcast/latest/sql/create-mapping.html>`__.
 
-See `IMDG Reference Manual
-<https://docs.hazelcast.com/imdg/latest/sql/querying-imap.html#key-and-value-fields>`__
-for details.
+Available connectors are:
 
-Lite Members
-~~~~~~~~~~~~
+- `Apache Kafka
+  <https://docs.hazelcast.com/hazelcast/latest/sql/kafka-connector.html>`__:
+  Read from and write to Kafka topics.
+- `File
+  <https://docs.hazelcast.com/hazelcast/latest/sql/file-connector.html>`__:
+  Read from a https://docs.hazelcast.com/hazelcast/latest/sql/connectors.htmllocal or remote file.
```

Comment:
again this unordered list does not look ok. It looks like this: 
![a](https://user-images.githubusercontent.com/32355782/133273401-e202cf52-d508-4863-a22f-577469990c13.png)